pkg/gce-cloud-provider/compute/gce-compute.go

```diff
@@ -459,6 +460,46 @@ func (cloud *CloudProvider) InsertDisk(ctx context.Context, project string, volK
 	}
 }
 
+func (cloud *CloudProvider) UpdateDisk(ctx context.Context, project string, volKey *meta.Key, existingDisk *CloudDisk, params common.ModifyVolumeParameters) error {
+	if volKey.Type() == meta.Regional {
+		return fmt.Errorf("Cannot update regional disk")
+	}
+	klog.V(5).Infof("Updating disk %v", volKey)
+	// hyperdisks are zonal disks
```

Comment:
// hyperdisks are zonal disks
// pd-disks do not support modification of IOPS and Throughput

Comment should go above `if volKey.Type() == meta.Regional {`

Reply:
Done